.github/SwiftCurrentLint/.swiftlint.yml

```diff
@@ -171,6 +171,6 @@ type_contents_order:
     ]
 
 file_header:
-    required_pattern: '//  Copyright © 2021 WWT and Tyler Thompson\. All rights reserved\.'
+    required_pattern: '//  Copyright © 202([0-9]) WWT and Tyler Thompson\. All rights reserved\.'
```

Comment:
I wonder if we should split this into an immediate PR  to `main` since it is more of a universal issue.

Reply:
See #162